src/wallet/wallet_tx_builder.h

```diff
@@ -239,6 +239,15 @@ enum class AddressResolutionError {
     RevealingReceiverAmountsNotAllowed,
 };
 
+class QuasiChangeError {
```

Comment:
I don't find this name intuitive; please add a comment explaining what it is (or change the name).

Reply:
I’ll document it (as I should have anyway), but will also rename it – how’s `PhantomChangeError` (it’s change that exists until you add an output for it, at which point it’s consumed by the increased fee).